### PR TITLE
Test account and comment validation

### PR DESCRIPTION
**Pull request by TheTexanCodeur**
This small PR tests hashcode and equality overrides that were not covered until now. 

Acceptance criteria: 

- [x] Test the hashcode override for the classes `CreateAccountValidation` and `NewCommentValidation`
- [x] Test the equality override for the classes `CreateAccountValidation` and `NewCommentValidation`

### CODE REVIEW

File: test/models/ui/validation/create_account_validation_test.dart [hunk 1]
```diff
@@ -35,6 +35,55 @@ void main() {
       );
 
       expect(createAccount, createAccountCopy);
+      expect(createAccount == createAccountCopy, true);
```
Comment (CHOOSEIT) [inline test/models/ui/validation/create_account_validation_test.dart:38]: I think that this line is a duplication of the line right above it .

File: test/models/ui/validation/new_comment_validation_test.dart [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+        posted: true,
+      );
+
+      final newCommentStateCopy = NewCommentValidation(
+        contentError: "commentError",
+        posted: true,
+      );
+
+      expect(newCommentState, newCommentStateCopy);
+      expect(newCommentState == newCommentStateCopy, true);
```
Comment (CHOOSEIT) [inline test/models/ui/validation/new_comment_validation_test.dart:36]: I think that is line is also a duplicate of the previous line.

**Review by CHOOSEIT — CHANGES_REQUESTED** [review]
Great work 💯 

Looks good to me.

I just have a two little comments.

Thank you 👍🏼

File: test/models/ui/validation/create_account_validation_test.dart (hunk 1 above)
Comment (TheTexanCodeur) [reply test/models/ui/validation/create_account_validation_test.dart:37]: Fixed with [5141c90](https://github.com/ProximaEPFL/proxima/pull/358/commits/5141c90560386380b3d26de95338fdc3622b7d00)

Comment (TheTexanCodeur) [reply test/models/ui/validation/create_account_validation_test.dart:37]: Fixed with [5141c90](https://github.com/ProximaEPFL/proxima/pull/358/commits/5141c90560386380b3d26de95338fdc3622b7d00)

File: test/models/ui/validation/new_comment_validation_test.dart (hunk 2 above)
Comment (TheTexanCodeur) [reply test/models/ui/validation/new_comment_validation_test.dart:35]: Fixed with [5141c90](https://github.com/ProximaEPFL/proxima/pull/358/commits/5141c90560386380b3d26de95338fdc3622b7d00)

**Review by CHOOSEIT — APPROVED** [review]
Looks good to me 👍🏼

**Pull request merged**